### DISCUSSION TITLE
Guess who's back!

### DISCUSSION THREAD

**Discussion by iminavo** [iminavo/scrum#13] [Announcements]
It's me, you guessed it, calm down.

Due to certain external circumstances, I had to retire for about 4 months and due to my laziness, I did not blog for another 4. I've generated about 400 lines of dirty notes but they are outdated now.

I've decided to continue this diary again, but first I have to provide you with some context of what has cordially changed, and the changes are significant.

Here l will briefly introduce you to the new _cast_, and the next one will be on processes.

While I was off the project, the deadlines were screwed /who would've thought/ and the teams and practices altered. Team leads, Dan and Mark, are no longer with us due to unrelated reasons. Sai is also gone, as are several devs that arrived just before I left. Vitar, the PM, and Karina, the BA, are no longer on the project as well.

Here's my current team:

* Petr Zapletal - my new team lead
* Oleg Kuznetsov - a /witty/ junior scala dev
* Sofia Huppmann -  a mid-scala dev
* Alexander J. Liu - a mid-scala dev

And some more people:

* Argen Alimbekov - the team lead of the second team
* Dave Gurnell - our new architect
* Ganesh Kumar - our new PM
* Ciara O'Flynn - our new BA
* Jose Manuel Fernandez - our new delivery coach

Those are the _main_ people I'm working with now, but there is a third team as well and a whole bunch of other people. I'll introduce them if they become relevant to the notes.

**Comment by f-person** [comment]
Hey! It's good to see you back <3

**Comment by SantiagoNeeringVQO** [comment]
Welcome back to your diary! It seems like there have been significant changes while you were away. It's great that you're getting back into it and providing context for your readers. I look forward to reading about your experiences with your new team and processes. Best of luck!

**Comment by ElveraGowersw0F** [comment]
My God! It's really great, I will continue to follow you guys.
